hudi-hadoop-common/src/test/java/org/apache/hudi/metadata/TestHoodieTableMetadataUtil.java

```diff
@@ -333,12 +334,14 @@ private static void writeParquetFile(String instant,
         path,
         metaClient.getStorage(),
         metaClient.getTableConfig(),
-        HoodieTestDataGenerator.AVRO_SCHEMA_WITH_METADATA_FIELDS,
+        HoodieSchema.fromAvroSchema(HoodieTestDataGenerator.AVRO_SCHEMA_WITH_METADATA_FIELDS),
```

Comment:
Similar to the other `HoodieSchema` you added to `HoodieTestDataGenerator`, can you update the class to have an equivalent `HoodieSchema` for `AVRO_SCHEMA_WITH_METADATA_FIELDS`?